LabController/src/bkr/labcontroller/proxy.py

```diff
@@ -237,7 +237,7 @@ def process_log(self, block):
                     self.logfile_name, create=(self.where == 0))
             with log_file:
                 log_file.update_chunk(block, self.where)
```

Comment:
My marching orders is to work on something else first and I'll squeeze this in.  Please be patient with my turnaround.